src/Base/Vector3D.cpp

```diff
@@ -227,6 +227,25 @@ bool Vector3<float_type>::IsEqual(const Vector3<float_type>& rclPnt, float_type
     return Distance(*this, rclPnt) <= tol;
 }
 
+template<class float_type>
+bool Vector3<float_type>::IsParallel(const Vector3<float_type>& rclPnt, float_type tol) const
+{

```

Comment:
The code has some issues:
* the argument is a vector and not a point, thus calling the parameter rclPnt is somewhat confusing
* the function doesn't handle the case if one if the vectors is the null vector
* it's not very efficient because the method Length() uses sqrt() internally.

Reply:
With the API of [Rhino](https://developer.rhino3d.com/api/rhinocommon/rhino.geometry.vector3d/isparallelto?version=8.x) it returns false if one of the vectors is the null vector.